### PR TITLE
fix: error handling for 2FAExpiredSession

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Fixes https://github.com/cds-snc/platform-forms-client/issues/2490

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [ ] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [x] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: public/static/locales/en/cognito-errors.json [hunk 1]
```diff
@@ -25,5 +25,9 @@
   "2FALockedOutSession": {
     "title": "Too many sign-in attempts ",
     "description": "Try signing in again to generate a new security code."
+  },
+  "2FASessionExpired": {
+    "title": "Session expired",
```
Comment (anikbrazeau) [inline public/static/locales/en/cognito-errors.json:30]:
```suggestion
    "title": "Your session expired",
```

File: public/static/locales/fr/cognito-errors.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "linkText": "Si vous n’avez pas de compte, créez-en un maintenant."
   },
   "2FALockedOutSession": {
```
Comment (craigzour) [inline public/static/locales/fr/cognito-errors.json:25]: I think you forgot to add a new key for `2FASessionExpired` in the French translation file

**Comment by timarney** [comment]
@anikbrazeau I fixed up the .json files with the right keys (was a mistake on my end) --- can you confirm we now have the right text?

**Review by craigzour — APPROVED** [review]
LGTM :) Let's see if the product team wants to include it or not in the upcoming release.

**Pull request merged**